added-basic-information-about-javascript-and-links (#36092)
freeCodeCamp/freeCodeCamp: guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/index.md

---
## Basic JavaScript

> Javascript (often abbreviated as "JS") , invented by "Brendan Eich" , is a high level,dynamic programming language which when combined with HTML can convert static websites into Dynamic Websites(having some functionality).It is one of the most versitile language of all time , finding applications in almost all the domains from Machine Learning to Data Scarping and Web Development , but most of its uses are in the Web Development.

Reviewer: ```suggestion
JavaScript (often abbreviated as "JS"), invented by "Brendan Eich", is a high level, dynamic programming language which when combined with HTML can convert Static Websites into Dynamic Websites (having some functionality). It is one of the most versatile languages of all time, finding applications in almost all domains from Machine Learning to Data Scraping and Web Development.
```
